content/en/dashboards/reporting.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Reporting
+kind: documentation
+disable_toc: false
+is_public: true
+beta: true
+is_beta: true
+further_reading:
+  - link: "https://www.datadoghq.com/blog/template-variable-associated-values/"
+    tag: "Blog"
+    text: "Use associated template variables to refine your dashboards"
+  - link: "https://learn.datadoghq.com/course/view.php?id=8"
+    tag: "Learning Center"
+    text: "Building Better Dashboards"
+---
+
+<div class="alert alert-warning">This feature is currently in private beta. To request access, reach out to support@datadoghq.com.</div>
+
+## Overview
+
+Reporting automatically sends a visual summary of a dashboard to selected recipients on a schedule. The generated reports display images of widgets from a dashboard in a linear HTML format.
+
+{{< img src="dashboards/reporting/report_example.png" alt="An example report titled Checkout KPI Report showing a date, a description, a dashboard link, and 3 images of dashboard widgets" style="width:100%;" >}}
+
+## Schedule a report
+
+Create a new report from any dashboard by opening the dashboard’s cog menu and selecting **Schedule a Report**.
+
+The following widget types are supported:
+
+- Change
+- Distribution
+- Geomap
+- Group
+- Heat Map
+- Notes and Links
+- Query Value
+- Scatter Plot
+- Service Summary
+- Table
+- Timeseries
+- Top List
+
+In the configuration modal that opens, set a schedule for the report to determine when and how often the report is sent. Set a time frame to determine the range of time displayed in the resulting report. The report time frame can be different from the time frame displayed on the dashboard.
+
+Add recipients to your report by entering their email addresses. The email associated with your Datadog account is automatically added as a recipient. You can remove yourself as a recipient by hovering over your email and clicking the red **X**. 
```

Comment:
Note that the X isn't actually red until the X itself is directly hovered over. I think it would be more accurate to describe the X as "the X next to the email address" (or something like that) instead of by color.

Reply:
Thanks, updated!